src/main/resources/tailormap-api.yaml

```diff
@@ -307,10 +324,150 @@ paths:
           required: true
           schema:
             type: integer
+            format: int64
       responses:
         '200':
           description: OK
           content:
             application/json:
               schema:
                 $ref: './model.yaml#/components/schemas/LayerDetails'
+
+  /features/{appid}/{applayerid}:
+    summary: 'Use this endpoint to access features.'
+    get:
+      description: 'retrieve features that fulfill the requested conditions (parameters).'
+      parameters:
+        - description: 'application id'
+          in: path
+          name: appid
```

Comment:
Although this is part of the path so the name does not really matter, I think it would be good to camelCase these as well? Same for applayerid below.